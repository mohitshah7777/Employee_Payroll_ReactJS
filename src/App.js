import React from "react";
import Register from "./pages/Register";
import Login from "./pages/Login";
import { BrowserRouter, Route, Switch } from 'react-router-dom'
require('dotenv').config()

function App() {
  return (
    <div className="App">
      <BrowserRouter>
        <Switch>
          <Route path='/login' component={Login} />
          <Route path='/register' component={Register} />
        </Switch>
      </BrowserRouter>
    </div>
  );
}

export default App;
