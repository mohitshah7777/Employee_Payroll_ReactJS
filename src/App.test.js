import { shallow } from 'enzyme';
import React from 'react';
import App from './App';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'
Enzyme.configure({ adapter: new Adapter() })

describe('App component', () => {
  test('Route is rendered properly', () => {
    const component = shallow(<App />)
    component.find('Route').exists()
    expect(component).toMatchSnapshot()
  })
})