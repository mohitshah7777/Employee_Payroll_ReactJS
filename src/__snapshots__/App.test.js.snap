// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Route is rendered properly 1`] = `ShallowWrapper {}`;
